### ISSUE TITLE
[Features] Do auto-play for the queue

### ISSUE THREAD

**Issue by cattaneod** [stegripe/rawon#630]
### Describe the solution you'd like

It would be nice to have an autoplay function that can be switched on or off.
If autoplay is activated, when the queue is finished, the bot should automatically add another song (related to the last one played) to the queue. This should be repeated as long as someone is in the voice chat.

If an user add a song to the queue, that song should be placed after the currently played one, and autoplay will trigger again when the queue is empty.

**Comment by Mednoob** [comment]
Is it like... when the queue is ended, the bot will auto-search another song that is related to the last song played in the queue?

**Comment by cattaneod** [comment]
Yes exactly, when the queue ends, the bot automatically search and add a related song to the queue

**Comment by Mednoob** [comment]
Autoplay would probably not be added in `2.3.0`. It will be added in `2.4.0`.

**Comment by mzrtamp** [comment]
Lazy de Dead.
